modules/storages/app/controllers/storages/admin/storages_controller.rb

```diff
@@ -162,6 +165,20 @@ def update
     end
   end
 
+  def change_health_email_notifications
+    return head :forbidden unless %w[1 0].include?(permitted_storage_params[:health_notifications_enabled])
```

Comment:
I think `forbidden` is semantically not correct here. What about 204: `head :no_content` or 400: `head :bad_request`?

Reply:
Good spot, yeah `bad_request` is the more semantic option